code/modules/overmap/exoplanets/decor/flora/potted.dm

```diff
@@ -71,10 +71,14 @@
 		return 1
 	return ..()
 
-// Added random icon selection for potted plants.
-// It was silly they always used the same sprite when we have 26 sprites of them in the icon file.
+/obj/structure/flora/pottedplant/random
+	name = "potted plant (DEPRECIATED)"
```

Comment:
```suggestion
	name = "potted plant (DEPRECATED)"
```
?

Reply:
the monetary value of potted plants decreases over time